### PR TITLE
fixed bug when export libraries

### PR DESCRIPTION
**Pull request by quantori-shishkin**

### CODE REVIEW

File: src/signals_notebook/materials/library.py [hunk 1]
```diff
@@ -366,15 +366,24 @@ def create_asset_with_batches(
...
+    def _is_file_ready(self, report_id: str) -> dict[str, Any]:
         api = SignalsNotebookApi.get_default_api()
         log.debug('Check job status for: %s| %s', self.__class__.__name__, self.eid)
 
         response = api.call(
             method='GET',
             path=(self._get_endpoint(), 'bulkExport', 'reports', report_id),
         )
-        return response.status_code == 200 and response.json()['data']['attributes']['status'] == 'COMPLETED'
+        response_attributes = response.json().get('data').get('attributes')
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/materials/library.py:377]: Are you sure this keys will be sent to response?

File: src/signals_notebook/materials/library.py [hunk 2]
```diff
@@ -411,7 +420,10 @@ def get_content(self, timeout: int = 30, period: int = 5) -> File:
         response = None
 
         while time.time() - initial_time < timeout:
-            if self._is_file_ready(report_id):
+            result = self._is_file_ready(report_id)
+            if result['error'] == 'Nothing to export.':
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/materials/library.py:424]: move it to Constants

File: src/signals_notebook/materials/library.py [hunk 3]
```diff
@@ -411,7 +420,10 @@ def get_content(self, timeout: int = 30, period: int = 5) -> File:
...
+                return File(name=f'{self.name}_empty', content=b'The library is empty', content_type='csv')
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/materials/library.py:425]: content_type isnt correct

Comment (quantori-cherednichenko) [reply src/signals_notebook/materials/library.py:425]: CSV is file extension
content_type should be written like 'smth/csv'

File: src/signals_notebook/materials/library.py (hunk 1 above)
Comment (quantori-shishkin) [reply src/signals_notebook/materials/library.py:377]: mm, yes. This keys u get always in all responses

**Review by quantori-cherednichenko — APPROVED** [review]

File: src/signals_notebook/materials/library.py [hunk 4]
```diff
@@ -411,7 +421,10 @@ def get_content(self, timeout: int = 30, period: int = 5) -> File:
         response = None
 
         while time.time() - initial_time < timeout:
-            if self._is_file_ready(report_id):
+            result = self._is_file_ready(report_id)
+            if result['error'] == EXPORT_ERROR_LIBRARY_EMPTY:
+                return File(name=f'{self.name}_empty', content=b'The library is empty', content_type='text/csv')
```
Comment (quantori-pokidovea) [inline src/signals_notebook/materials/library.py:426]: This is invalid CSV file

**Review by quantori-pokidovea — APPROVED** [review]

**Pull request merged**